mscore/musescore.cpp

```diff
@@ -7720,6 +7720,16 @@ int runApplication(int& argc, char** av)
       return qApp->exec();
       }
 
+//---------------------------------------------------------
+//   showSplashMessage
+//---------------------------------------------------------
+
+inline static void showSplashMessage(QSplashScreen* sc, QString&& message)
+      {
+      if (sc)
+            sc->showMessage(message, Qt::AlignHCenter | Qt::AlignBottom, Qt::white);
```

Comment:
Wouldn't it make sense to print those messages to stdout (or as a `QInfo()`) in case of `noGui` (and thus no `sc`)?

Reply:
Done.